lottie-swift/src/Public/iOS/Compatibility/CompatibleAnimationView.swift

```diff
@@ -0,0 +1,270 @@
+//
+//  CompatibleAnimationView.swift
+//  Lottie_iOS
+//
+//  Created by Tyler Hedrick on 3/6/19.
+//
+
+import Foundation
+import UIKit
+
+@objc
+/// An Objective-C compatible wrapper around Lottie's Animation class.
+/// Use in tandem with CompatibleAnimationView when using Lottie in Objective-C
+public final class CompatibleAnimation: NSObject {
+
+  @objc
+  static func named(_ name: String) -> CompatibleAnimation {
+    return CompatibleAnimation(name: name)
+  }
+
+  @objc
+  public init(name: String, bundle: Bundle = Bundle.main) {
+    self.name = name
+    self.bundle = bundle
+    super.init()
+  }
+
+  internal var animation: Animation? {
+    return Animation.named(name, bundle: bundle)
+  }
+
+  // MARK: Private
+
+  private let name: String
+  private let bundle: Bundle
+}
+
+@objc
+/// An Objective-C compatible wrapper around Lottie's AnimationView.
+public final class CompatibleAnimationView: UIView {
+
+  @objc
+  init(compatibleAnimation: CompatibleAnimation) {
+    animationView = AnimationView(animation: compatibleAnimation.animation)
+    super.init(frame: .zero)
+    commonInit()
+  }
+
+  @objc
+  public override init(frame: CGRect) {
+    animationView = AnimationView()
+    super.init(frame: frame)
+    commonInit()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
```

Comment:
No support for using `CompatibleAnimationView` in a storyboard/xib? I'm guessing it won't be any different of an initializer than `init(frame:)`.